src/components/Sidebar.tsx

```diff
@@ -34,39 +35,39 @@ const Sidebar: React.FC = () => {
           active.type == EntityType.TransactionTemplate ? active.index : null
         }
         onSelect={(_, id) => {
-          navigate(`/${projectPath}?type=tx&id=${id}`)
+          navigate(`/${projectPath}?type=tx&id=${id}&storage=${selectedResourceAccount || 'none'}`)
```

Comment:
We can calculate and store value of `selectedResourceAccount || 'none'` in a single place to DRY it a bit.

Reply:
Changed in [this commit](https://github.com/onflow/flow-playground/pull/175/commits/289b5d04ba186213347bb27b87021fdfc8efaf3e)